app/services/mandataris.js

```diff
@@ -143,20 +146,15 @@ export default class MandatarisService extends Service {
     };
   }
 
-  async isMandatarisActive(mandataris) {
-    // assuming that there always be a folded mandataris for the mandataris
-    const foldedMandataris = (await fold([mandataris])).at(0);
-    const now = moment();
-    const todayIsInBetweenPeriod =
-      moment(foldedMandataris.foldedStart).isBefore(now) &&
-      (!foldedMandataris.foldedEnd ||
-        moment(foldedMandataris.foldedEnd).isAfter(now));
-
-    if (!todayIsInBetweenPeriod) {
-      return false;
-    }
-
-    const status = await foldedMandataris.mandataris.status;
-    return status && status.uri !== MANDATARIS_BEEINDIGD_STATE;
+  async updateCurrentFractie(mandatarisId) {
+    const mandataris = await this.store.findRecord('mandataris', mandatarisId);
+    const persoon = await mandataris.isBestuurlijkeAliasVan;
+    const bestuursperiode = await this.mandatarisRepository.getBestuursperiode(
+      mandataris.id
+    );
+    await this.persoonRepository.updateCurrentFractie(
```

Comment:
Now I get why you fetch the bestuursperiode, because it makes more sense to call the mandataris service with a person instead of a mandataris, but I still think it makes more sense to just call the mandataris service just with the mandataris. (or maybe mandataris and person if you want).